velox/functions/prestosql/tests/ProbabilityTest.cpp

```diff
@@ -155,5 +155,36 @@ TEST_F(ProbabilityTest, binomialCDF) {
       binomialCDF(-2, 0.5, -1), "numberOfTrials must be greater than 0");
 }
 
+TEST_F(ProbabilityTest, laplaceCDF) {
+  const auto laplaceCDF = [&](std::optional<double> location,
+                              std::optional<double> scale,
+                              std::optional<double> x) {
+    return evaluateOnce<double>("laplace_cdf(c0, c1, c2)", location, scale, x);
+  };
+
+  // Test cases
```

Comment:
Please add inputs with std::nullopt, doubleMax, doubleMin values

Reply:
Thanks for your review @aditi-pandit , UT cases added